ui-packages/packages/trusty/src/components/Organisms/ModelDiagram/ModelDiagram.tsx

```diff
@@ -14,49 +13,31 @@ const ModelDiagram = (props: ModelDiagramProps) => {
   const { model } = props;
   const type: string = model.type;
 
-  const editorEnvelopeLocator: EditorEnvelopeLocator = {
-    targetOrigin: window.location.origin,
-    mapping: new Map([
-      [
-        'dmn',
-        {
-          resourcesPathPrefix: '../gwt-editors/dmn',
-          envelopePath: '/envelope/envelope.html'
-        }
-      ]
-    ])
-  };
-
-  if (type === DMN1_2) {
-    return makeDMNEditor(model, editorEnvelopeLocator);
-  }
-
-  return DEFAULT;
+  useEffect(() => {
+    let editor: StandaloneEditorApi | undefined = undefined;
+    if (type === DMN1_2) {
+      editor = DmnEditor.open({
+        container: document.getElementById('dmn-editor-container'),
+        initialContent: Promise.resolve(model.model),
+        readOnly: true
+      });
+    }
+    return () => {
+      if (editor) {
+        editor.close();
+      }
+    };
```

Comment:
You can actually use return the cleanup function of this `useEffect` inside the if. The reasoning is that when the Editor is created, we tell React to close it when cleaning up. When the Editor is not created (`type !== DMN1_2`), we don't need to cleanup anything. So:
```suggestion
    if (type === DMN1_2) {
      const editor = DmnEditor.open({
        container: document.getElementById('dmn-editor-container'),
        initialContent: Promise.resolve(model.model),
        readOnly: true
      });
      
      return () => {
        editor.close();
      }
```

Reply:
Approving as this is optional.

Reply:
@tiagobento Update made. Thank-you (and thank-you for approving).